tests/test_lca.py

```diff
@@ -69,6 +69,22 @@ def test_api_create_insert_bad_ksize():
         lca_db.insert(ss)
 
 
+def test_api_create_insert_bad_ident():
+    # can we insert a ksize=21 signature into a ksize=31 DB? hopefully not.
```

Comment:
```suggestion
    # can we insert a signature with no ident?
```

(I assume this test is meant to pass?) No assertion here...

Reply:
yep. adding comment in code. thx :)